bitnami/metallb/values.yaml

```diff
@@ -0,0 +1,230 @@
+## Default values for metallb.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
```

Comment:
As I commented, it is important for our CI/CD to add the globals.

Reply:
If the `storageClass` is not needed it is fine.

Reply:
done